### PR TITLE
Quay onboarding initial draft

### PR DESCRIPTION
**Pull request by vishnuchalla**
## Type of change

- [x] Refactor
- [x] New feature
- [ ] Bug fix
- [x] Optimization
- [ ] Documentation Update

## Description
On-boarding Quay team performance test results to the UI Dashboard. Did some refactoring and optimization along the line.

## Related Tickets & Documents

- Related Issue #
- Closes #

## Checklist before requesting a review

- [x] I have performed a self-review of my code.
- [ ] If it is a core feature, I have added thorough tests.

## Testing
- Tested in local by spinning up the UI using `local-compose.sh` script.
Let me know if you need me to share credentials to play-around with Quay stuff.

### CODE REVIEW

**Review by chentex — CHANGES_REQUESTED** [review]
I just have one comment on the PR after testing it locally.
Thank you for the refactor and making it more generic!

File: backend/app/api/v1/endpoints/quay/quayGraphs.py [hunk 1]
```diff
@@ -0,0 +1,319 @@
...
+    print(query)
+    es = ElasticService(configpath="quay.elasticsearch",index=index)
+    results = await es.post(query,size=0)
+    await es.close()
+    return results
+
+
+async def getQuayMetrics(uuids: list, index: str):
+    ids = "\" OR uuid: \"".join(uuids)
+    print(ids)
```
Comment (jtaleric) [inline backend/app/api/v1/endpoints/quay/quayGraphs.py:188]: We print the query and the ids here... can we just print one or the other? (the query should suffice?)

**Review by jtaleric — APPROVED** [review]
overall lgtm! small mention

File: backend/app/api/v1/endpoints/cpt/cptJobs.py [hunk 2]
```diff
@@ -42,7 +44,7 @@ async def jobs(start_date: date = Query(None, description="Start date for search
     for product in products:
         try:
             df = await products[product](start_date, end_date)
```
[the remark this answers is not in the document]
Comment (vishnuchalla) [reply backend/app/api/v1/endpoints/cpt/cptJobs.py:46]: Changed as suggested.

File: backend/app/api/v1/endpoints/quay/quayGraphs.py (hunk 1 above)
Comment (vishnuchalla) [reply backend/app/api/v1/endpoints/quay/quayGraphs.py:188]: Removed ids.

**Review by chentex — APPROVED** [review]
LGTM, thanks!

**Pull request merged**